packages/devtools_app/lib/src/utils.dart

```diff
@@ -1069,3 +1069,159 @@ String prettyTimestamp(
   );
   return DateFormat.Hms().format(timestampDT); // HH:mm:ss
 }
+
+/// A [ChangeNotifier] that holds a list of data.
+///
+/// This class also exposes methods to interact with the data. By default,
+/// listeners are notified whenever the data is modified, but notifying can be
+/// optionally disabled.
+class ListValueNotifier<T> extends ChangeNotifier
+    implements ValueListenable<List<T>> {
+  /// Creates a [ListValueNotifier] that wraps this value [_rawList].
+  ListValueNotifier(List<T> rawList) : _rawList = List<T>.from(rawList) {
+    _currentList = ImmutableList(_rawList);
+  }
+
+  List<T> _rawList;
+
+  ImmutableList<T> _currentList;
+
+  @override
+  List<T> get value => _currentList;
+
+  void add(T item, {bool notify = true}) {
```

Comment:
Add a comment that `notify=false` should rarely be used or remove it.

Reply:
removed notify param